packages/abstract-document/src/abstract-document-exporters/pdf/render.ts

```diff
@@ -413,16 +423,21 @@ function drawText(
   text: string,
   alignment: AD.TextStyle.TextAlignment
 ) {
+  //compensation needed as pdfKit's widthOfString may return a slightly
+  //lower value than the actual size due to loss of precision
+  const compensatePdfKitSize = 1;
```

Comment:
Is it really necessary to pass compensatePdfKitSize to calculateTextAlignmentOffset function? Maybe it's better to put this inside calculateTextAlignmentOffset since it's more or less constant when everything works.

Reply:
The idea was to show all those places uses the same constant for a reason, it would make it easier to debug in the future if you know why theres a constant there, and if we need to increase it to + 2 then there's fewer places to edit